05_run_ocp.sh

```diff
@@ -124,15 +124,15 @@ sudo systemctl reload NetworkManager
 cat /tmp/dnsmasq.conf | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=dnsmasq.conf
 
 # Build and start the ironic container
-cat ironic/runironic.sh | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=runironic.sh
-cat ironic/Dockerfile | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=Dockerfile
-ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman build \
-    --build-arg RHCOS_IMAGE_URL=${RHCOS_IMAGE_URL} \
-    --build-arg RHCOS_IMAGE_VERSION=${RHCOS_IMAGE_VERSION} \
-    --build-arg RHCOS_IMAGE_FILENAME_OPENSTACK=${RHCOS_IMAGE_FILENAME_OPENSTACK} \
-    -t ironic:latest .
+cat ${RHCOS_IMAGE_FILENAME_OPENSTACK}.gz | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=${RHCOS_IMAGE_FILENAME_OPENSTACK}.gz
+ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman pull docker.io/vrutkovs/metalkube-ironic
```

Comment:
Can we get this setup to use the metalkube quay repo @russellb mentioned?

Reply:
Yes, could someone upload this image there?

Reply:
Should we wait until we are triggering rebuilding the image before merging this

Reply:
Updated to pull https://quay.io/repository/metalkube/metalkube-ironic, which is now being built from https://github.com/metalkube/metalkube-ironic